test/verify/check-system-shutdown-restart

```diff
@@ -152,11 +153,31 @@ class TestShutdownRestart(MachineCase):
         b.wait_not_val("#shutdown-time-input", "")
         old = b.val("#shutdown-time-input")
         b.set_input_text("#shutdown-time-input", "blah")
+        b.wait_not_present("#delay-helper")
```

Comment:
Meh, this is still [causing a race condition](https://logs.cockpit-project.org/logs/pull-16053-20210707-072202-f3bae295-rhel-8-5/backup.6fl6fuzc/log.html#283). It's not that bad, and the parallel test run that got triggered may just get green. In the light of this being release-critical, this can be a follow-up.

Reply:
It [failed also on firefox](https://logs.cockpit-project.org/logs/pull-16053-20210707-072202-f3bae295-fedora-34-firefox/log.html#283-1), so hard to get green. I can reproduce this locally. It means that the date validation function did not run correctly, as 07/07/2021 (as in the screenshot) is clearly correct. So my suspicion is that the delay-helper was there all along, not triggered by the "blah" input.

Reply:
Pushed a fixup which *may* help. I've run the test a dozen times locally, and don't get the failure any more (but may just be good/bad luck)